.github/workflows/nightly.yml

```diff
@@ -163,7 +170,8 @@ jobs:
 
   notify_failed:
     runs-on: ubuntu-latest
-    if: always() && contains(needs.*.result, 'failure')
+    # Do not run on dry_run or success
+    if: ${{ !(inputs.dry_run) }} && contains(needs.*.result, 'failure')
```

Comment:
Don't you have to put `always` anyway here so that GH won't skip this job if dependencies fail? I'm not knowledgeable enough to be certain here.

Reply:
I will add it then